modules/Microsoft.Sql/servers/.test/secondary/dependencies.bicep

```diff
@@ -0,0 +1,38 @@
+@description('Required. The name of the server.')
+param serverName string
+
+@description('Required. The name of the database.')
+param databaseName string
+
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'sqlsec'
```

Comment:
I would suggest to keep the serviceShort in the main test file and pass the server name down - as we do for the other tests.

Reply:
Yup, that was a slip on my end. Removed it now.